.github/workflows/deploy-eb.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy EB Prod
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    types: manual-trigger
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    - name: Build and push docker image
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        push: true
+        tags: serotracker/iit-backend:latest
+        file: Dockerfile-app
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8.3'
+    - name: Install aws eb cli
+      run: |
+        pip install awsebcli
+    - name: Deploy Elastic Beanstalk App
+      run: |
+        eb deploy iit-backend-prod
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
What about the three final sections that you included in the screenshot in basecamp? Post Build and push docker image, Post Login to Dockerhub, Complete job

Reply:
Those steps are included in part of the "docker/build-push-action@v2" above